README_ENV.md

```diff
@@ -31,6 +32,15 @@ The `TZ` setting affects logging output and the backup function. [TZ identifiers
 
 The `BACKUP_CRON_EXPRESSION` setting affects the backup function. In a Cron-Expression, you define an interval for when to run jobs. This image uses Supercronic for crons, see https://github.com/aptible/supercronic#crontab-format or https://crontab-generator.org
 
+#### Remote Control
+If you enable the container setting the Container will not autostart the gameserver, it will however provide webhooks that allow you to control the server remotely. Hooks provided are
```

Comment:
Which setting again? Im a new user, why isnt this desribed here, its a readme right?